### ISSUE TITLE
MA0124 can't handle named tuples

### ISSUE THREAD

**Issue by FreeYourMinds** [meziantou/Meziantou.Analyzer#626]
### Version of the Meziantou.Analyzer NuGet package

2.0.98

### Rule Identifier

MA0124

### Target Framework

net6;net7

### C# Language version

C# 11

### Description

# MA0124 can't handle named tuples

When using C# 7 named tuples, MA0124 fails to match the type

## LoggerParameterTypes.txt

```
@ItemQuantities;System.Collections.Generic.IList{System.ValueTuple{System.Guid,System.Int32}}
```

### Error Reported

> Error MA0124 : Parameter '@ItemQuantities' must be of type 'System.Collections.Generic.IList<(System.Guid, int)>' but is of type 'System.Collections.Generic.IList<(System.Guid ItemId, int Quantity)>'

*Note: severity was intentially set to error*

### Reproduction Steps

Minimal code:

```c#
IList<(Guid ItemId, int Quantity)> itemsList = new List<(Guid, int)>
{
	(Guid.NewGuid(), 7),
	(Guid.NewGuid(), 2),
	(Guid.NewGuid(), 4),	
};

// this causes MA0124 to report
this.logger.LogInformation("Collected items: {@ItemQuantities}", itemsList);

// this is accepted flawlessly
this.logger.LogInformation("Collected items: {@ItemQuantities}", (IList<(Guid, int)>)itemsList);
```
